aspnetcore/test/debug.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Debug ASP.NET Core apps
```

Comment:
Not true.

Reply:
I didn't look in detail at the links ... was just going off of the issue title.

Since this is broader, then perhaps combining with *Troubleshooting* into an *Overview*. Just a passing 🤔.